Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationResultDispatchingValidator.cs

```diff
@@ -216,10 +217,16 @@ protected override bool ControlPropertiesValid ()
           {
             if (baseValidator is not IBusinessObjectBoundEditableWebControlValidationResultDispatcher && !baseValidator.IsValid)
             {
+              var property =
+                  ((BocSimpleColumnDefinition)columnDefinitions[0])
+                  .GetPropertyPath()
+                  .Properties
+                  .Single(() => new InvalidOperationException("No property found on column definition in edit mode"));
```

Comment:
This single implementation is an extension method from re-motion suggested to use by @MichaelKetting. 

I think providing a special message for the other case as well would probably be a good idea. I had an implemention with Asserts, do you think that would be a good idea?

Reply:
Why just a custom exception for the empty case?
Because it is the most likely error that happens is the case of no elements. In this case you want to have information what happened (e.g. details on the operation that resulted in the empty sequence). For more-than-one item, that's usually a much unlikely case or a true bug, not just an unexpected occurance.

Could we also do two exception?
We could, not sure if it's helpful on most cases. Nobody needed it by now, but we can add it if we want.

Assertions: it's what we do if we don't have other options. So, if there's just one or two instances in the code base that also need to cover multiple results, the assertion might be easiest, in terms of the API.